project_event/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = ['res.partner']
+
+    client_type = fields.Many2one(
```

Comment:
@jananjoy this field is added but it is not used

Reply:
@rimbendhaou  :As per new changed , the field named is changed to category_id